docs/configuration.html

```diff
@@ -303,6 +303,11 @@ <h4><a id="org.apache.kafka.disallowed.login.modules"></a><a id="systempropertie
       </tbody></table>
     </li>
   </ul>
+
+  <h3 class="anchor-heading"><a id="remotestorageconfigs" class="anchor-link"></a><a href="#remotestorageconfigs">3.10 Remote Storage Configs</a></h3>
```

Comment:
This feature is known as "Tiered Storage". Let us not change that as it will cause confusion to users.